.github/actions/install-openssl/action.yml

```diff
@@ -32,7 +32,7 @@ runs:
       if: runner.os == 'Windows'
       shell: bash
       run: |
-        choco upgrade openssl --no-progress
+        choco install openssl --version 3.1.1 -y --no-progress
```

Comment:
I feel like we had to use upgrade before because it would complain that it was already installed. But now it seems happy. /shrug.

Reply:
Yea, looks like it succeeded.